README.md

```diff
@@ -264,27 +270,97 @@ headers.set('foobar', 'buzz')
 headers.get('foobar') // -> 'buzz'
 ```
 
+#### `Headers.values()`
+
+Returns: `IteratableIterator<string>`
+
+Each iteration of the headers `values()` iterator yields a header _value_
+
+```js
+const headers = new Headers()
+
+headers.set('abc', '123')
+headers.set('def', '456')
+headers.set('ghi', '789')
+headers.append('ghi', '012')
+
+for (const value of headers.values()) {
+	console.log(value)
+}
+
+// -> '123'
+// -> '456'
+// -> '789, 012'
+```
+
+#### `Headers.keys()`
+
+Returns: `IteratableIterator<string>`
+
+Each iteration of the headers `keys()` iterator yields a header _name_
+
+```js
+const headers = new Headers()
+
+headers.set('abc', '123')
+headers.set('def', '456')
+headers.set('ghi', '789')
+headers.append('ghi', '012')
+
+for (const name of headers.keys()) {
+	console.log(name)
+}
+
+// -> 'abc'
+// -> 'def'
+// -> 'ghi'
+```
+#### `Headers.forEach(callback, [thisArg])`
```

Comment:
```suggestion
#### `Headers.forEach(callback, [thisArg])`

- callback `(value: string, key: string, iterable: Headers) => void`
- thisArg `any` (optional)
```

Reply:
Gotcha, should've looked at the other parameterized methods - kept * style as with the others.